web/src/main/java/com/google/sps/servlets/SaveStockServlet.java

```diff
@@ -47,6 +49,41 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     long timestamp = System.currentTimeMillis();
 
+    Date date = Calendar.getInstance().getTime();
+    String gooddate = date.toString();
+    String[] date_ok = gooddate.split(" ");
+
+    if (date_ok[0].equalsIgnoreCase("Sun")) {
+      date_ok[0] = "Sunday";
+    } else if (date_ok[0].equalsIgnoreCase("Mon")) {
+      date_ok[0] = "Monday";
+    } else if (date_ok[0].equalsIgnoreCase("Tue")) {
+      date_ok[0] = "Tuesday";
+    } else if (date_ok[0].equalsIgnoreCase("Wed")) {
+      date_ok[0] = "Wednesday";
+    } else if (date_ok[0].equalsIgnoreCase("Thu")) {
+      date_ok[0] = "Thursday";
+    } else if (date_ok[0].equalsIgnoreCase("Fri")) {
+      date_ok[0] = "Friday";
+    } else {
+      date_ok[0] = "Saturday";
+    }
+    if (date_ok[1].equalsIgnoreCase("Mar")) {
+      date_ok[1] = "March";
+    }
+
+    StringBuilder exactdate = new StringBuilder();
+    exactdate.append(date_ok[0]);
+    exactdate.append(", ");
+    exactdate.append(date_ok[1] + " ");
+    exactdate.append(date_ok[2]);
+    exactdate.append(", ");
+    exactdate.append(date_ok[5]);
+    exactdate.append(" at ");
+    exactdate.append(date_ok[3]);
+
+    String printdate = exactdate.toString();
```

Comment:
This isn't a bad idea but I'd like to know what you will be doing with this information? Also camel case printDate.

Reply:
![image](https://user-images.githubusercontent.com/20563330/112003078-f2801400-8af6-11eb-8ba1-3c7124f25bfe.png)
So that we can show the date and time when we hover over points so it should show that ^ instead of 24 here v
![image](https://user-images.githubusercontent.com/20563330/112003259-1f342b80-8af7-11eb-8aa7-5438a61e886a.png)